lms/djangoapps/support/static/support/jsx/errors_list.jsx

```diff
@@ -0,0 +1,34 @@
+/* global gettext */
+/* eslint react/no-array-index-key: 0 */
```

Comment:
Why disable this rule? You should add a key for your list items to ensure that React can detect only the changed elements. This [page](indentation) is a good place to read about keys and best practices for specifying keys (e.g. don't use the array index, etc.).

Reply:
I disabled this because we don't have any keys to add in different errors. Secondly I can't access the link you provided. https://github.com/edx/edx-platform/pull/indentation